src/constants/label.js

```diff
@@ -1,9 +1,24 @@
 "use strict";
 
 // Name fields in order of preference
-export const name_en = [
-  "coalesce",
-  ["get", "name:en"],
-  ["get", "name:latin"],
-  ["get", "name"],
-];
+export const localizedName = (function () {
+  let userLocales =
+    "languages" in navigator ? [...navigator.languages] : [navigator.language];
```

Comment:
```suggestion
  let userLocales = navigator.languages ?? [navigator.language];
```

Reply:
Thanks, this didn’t work when I first tried it because I hadn’t split out a separate `locales` array yet. 👍